Parse/build.gradle

```diff
@@ -41,12 +41,17 @@ android {
 
 ext.okhttpVersion = '3.6.0'
 dependencies {
+    compile 'com.android.support:support-annotations:25.3.1'
```

Comment:
the support version we use should be an ext { } definition likely in the root build.gradle.  this way the starter project that you updated can also just pull from this version